lstchain/reco/volume_reducer.py

```diff
@@ -0,0 +1,146 @@
+    :param config: [dict]
+            Loaded configuration: from where the algorithm (volume reduction method)
+            and their corresponding parameters are loaded.
+
+    :return: none.
+            Modifies the event containers by applying the computed mask to the image,
+            the waveform and the pulse_time objects, as:
+            image[~mask] = 0, ...
+    """
+    camera = event.inst.subarray.tel[tel_id].camera
+    image = event.dl1.tel[tel_id].image
+    pulse_time = event.dl1.tel[tel_id].pulse_time
+    waveform = event.r1.tel[tel_id].waveform
+
+    if config['volume_reducer']['algorithm'] is None:
+        algorithm = none_volume_reduction
+        parameters = config['volume_reducer']['parameters']
+    else:
+        algorithm = globals()[config['volume_reducer']['algorithm']]
+        parameters = config['volume_reducer']['parameters']
+
+    print("Volume reduction algorithm:", algorithm)
+
+    mask_volume_reduction = algorithm(image, camera, **parameters)
+    image[~mask_volume_reduction] = 0
+    waveform[:, ~mask_volume_reduction, :] = 0
+    pulse_time[~mask_volume_reduction] = 0
+
+
+def zero_suppression_tailcut_dilation(image, camera_geometry, **kwargs):
+    """
+    `tailcut_clean` + 3 `dilate` volume reducer method.
+
+    Default parameters for the tailcut_clean:
+        'picture_thresh': 8
+        'boundary_thresh' 4
+        'keep_isolated_pixels', True
+        'min_number_picture_neighbors', 0
+
+    :param image: [array]
+            Pixel values. A `event.inst.subarray.tel[telescope_id].camera` object.
+    :param camera_geometry: [Camera geometry information]
+            `ctapipe.instrument.CameraGeometry` object.
+    :param kwargs: [dict]
+            A dictionary containing the parameters of the selected volume reducer algorithm.
+
+    :return:[array]
+            A boolean mask (array) that contains the zero suppressed pixels.
+    """
+    kwargs.setdefault('picture_thresh', 8)
+    kwargs.setdefault('boundary_thresh', 4)
+    kwargs.setdefault('keep_isolated_pixels', True)
+    kwargs.setdefault('min_number_picture_neighbors', 0)
+
+    picture_zs = kwargs['picture_thresh']
+    boundary_zs = kwargs['boundary_thresh']
+    keep_isolated_pix_zs = kwargs['keep_isolated_pixels']
+    min_num_pict_neigh_zs = kwargs['min_number_picture_neighbors']
+
+    mask_0_suppression = tailcuts_clean(camera_geometry,
+                                        image,
+                                        picture_thresh=picture_zs,
+                                        boundary_thresh=boundary_zs,
+                                        keep_isolated_pixels=keep_isolated_pix_zs,
+                                        min_number_picture_neighbors=min_num_pict_neigh_zs
+                                        )
+    mask_pixel_0_suppressed = dilate_pixel_mask(mask_0_suppression, camera_geometry)
+
+    return mask_pixel_0_suppressed
+
+
+def dilate_pixel_mask(pixel_mask, camera_geometry):
```

Comment:
This is just a matter of taste but I'd have kept the order of the arguments of other functions (`dilate`, `tailcut_cleaning`...): `(geom, pixel_mask)`
(available for other functions)